tests/test_data.py

```diff
@@ -1311,63 +1311,88 @@ def test_load_chmap(self):
         self.assertEqual(acq_ch_position.shape[1], 2)
 
     def test_align_recording_drives(self):
-        # Create plot for documentation with affi and biegnet
+        # Create plots for documentation with affi and biegnet
         fig, ax = plt.subplots(1,2)
         subjects = ['affi', 'beignet']
         neuropixel_drive='NP_Insert137'
         drive2 = 'ECoG244'
+        drive2_ch_pos, _ , _ = load_chmap(drive_type=drive2)
+        drive2_offset = -(np.max(drive2_ch_pos, axis=0) + np.min(drive2_ch_pos, axis=0))/2 
```

Comment:
these offsets are pretty unintuitive. centering the maps by default would simplify things a lot